cpp/src/parquet/column_writer.cc

```diff
@@ -154,6 +157,8 @@ inline const T* AddIfNotNull(const T* base, int64_t offset) {
   return nullptr;
 }
 
+constexpr int64_t kHashBatchSize = 256;
```

Comment:
Should we move it to bloom_filter.h?

Reply:
I think it's a writer-side logic currently?